pkg/apis/dmi/services.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2022 The KubeEdge Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package dmi
+
+import (
+	dmiapi "github.com/kubeedge/kubeedge/pkg/apis/dmi/v1alpha1"
+)
+
+// DeviceManagerService defines the public APIS for remote device management.
+// The server is implemented by the module of device manager in edgecore
+// and the client is implemented by the device mapper for upstreaming.
+type DeviceManagerService interface {
+	// MapperRegister register the information of the mapper to device manager.
+	MapperRegister(*dmiapi.MapperRegisterRequest) (*dmiapi.MapperRegisterResponse, error)
+	// ReportDeviceStatus report the status of devices to device manager.
+	ReportDeviceStatus(*dmiapi.ReportDeviceStatusRequest) (*dmiapi.ReportDeviceStatusResponse, error)
+}
+
+// DeviceMapperService defines the public APIS for remote device management.
+// The server is implemented by the device mapper
+// and the client is implemented by the module of device manager in edgecore for downstreaming.
+type DeviceMapperService interface {
+	// CreateDevice register a device to the device mapper
```

Comment:
The comment in this file should be consitant with the api.proto

Reply:
ok. i have fixed this problem.